game.js

```diff
@@ -380,11 +387,22 @@ function main() {
   });
 
   const game = new Game({
-    width: 25,
-    height: 25,
-    squareSize: 20,
+    width: WIDTH,
+    height: HEIGHT,
+    squareSize: SQUARE_SIZE,
     players: [player1, player2],
   });
 
-  game.start();
+  const table = document.getElementById("table-stats");
```

Comment:
i shouldn't be seeing any `document.getElementById` calls here. That is way too low level. Please work on the same abstraction layer